discovery/discovery_test.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
```

Comment:
slick name. this makes me want to do `go.livepeer.org/whatever` for purely cosmetic reasons

Reply:
I like the idea 😃

Reply:
actually we totally should, looks like it's as easy as https://github.com/GoogleCloudPlatform/govanityurls

Reply:
Now we need package to publish 😃